CS342MemberViews/Package.swift

```diff
@@ -63,6 +64,19 @@ let package = Package(
             resources: [
                 .process("Resources")
             ]
+        ),
+        .target(
+            name: "SonyaJin",
+            dependencies: [
+                .target(name: "InstructorViews")
+            ],
+            exclude: [
+                "Resources/SonyaJin.jpeg.license",
+                "Resources/SonyaJin.md.license"
+            ],
```

Comment:
You get an error message that you having an invalid `exclude` statement. 
You are trying to exclude the _"Resources/SonyaJin.jpeg.license"_ and _"Resources/SonyaJin.md.license"_ files but you have not added any of these files to the repo. Please add these files including a proper license so your REUSE checks are passing. You can find out more about the REUSE and SwiftLint checks here: https://reuse.software/spec/ . Please refer to the other Swift Package targets and files to see the typical license header that we are using.